package.json

```diff
@@ -19,11 +19,15 @@
   },
   "description": "",
   "devDependencies": {
+    "@types/enzyme": "^3.10.4",
+    "@types/enzyme-adapter-react-16": "^1.0.5",
     "@types/jest": "24.9.1",
     "@types/nanoid": "2.1.0",
     "@types/react": "16.9.19",
     "@types/react-color": "3.0.1",
     "@types/react-dom": "16.9.5",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
```

Comment:
can you pin the versions?

Reply:
did you run npm install afterwards?

Reply:
Yes, for some reason the package-lock.json wasn't updated, right? Anyway, I think is working 
Let me know if you think we should do something else!!

Reply:
From my side it's all good.. maybe in the following PRs, we could add an option to clear the list (as a separate item in the dropdown) but it's good for now. Let's see what the others will say. Thanks!